pom.xml

```diff
@@ -2392,7 +2392,7 @@
 
         <!-- Identity REST API feature -->
         <identity.api.dispatcher.version>2.0.13</identity.api.dispatcher.version>
-        <identity.user.api.version>1.3.18</identity.user.api.version>
+        <identity.user.api.version>1.3.19-SNAPSHOT</identity.user.api.version>
```

Comment:
should update the version after merging https://github.com/wso2/identity-api-user/pull/180